2025-02-XX-novacustom-v54-series-qubes-certified.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title: "The NovaCustom V54 Series 14.0 inch coreboot laptop is Qubes certified!"
+categories: announcements
+image: /attachment/site/novacustom-V54-series.png
+author: The NovaCustom and Qubes teams
+---
+
+It is our pleasure to announce that the [NovaCustom V54 Series 14.0 inch coreboot laptop](https://novacustom.com/product/V54-series/) is [officially certified](/doc/certified-hardware/) for Qubes OS Release 4!
+
+## V54 Series 14.0 inch coreboot laptop
+
+Introducing the latest coreboot laptop equipped with cutting-edge technology. With an ultra-efficient 14th-gen Intel Meteor Lake CPU, a robust 73 WH battery, and a captivating 16:10 screen, your productivity will soar to new heights. Customize your device with a maximum of 96 GB DDR5 RAM and up to two lightning-fast PCIe SSDs. The Dasharo coreboot firmware ensures a reliable and secure foundation. Additionally, enjoy a variety of connectivity options such as Thunderbolt 4, Ethernet, plentiful USB ports, and optional Wi-Fi 7.
+
+[![Photo of the NovaCustom V54 Series 14.0 inch coreboot laptop](/attachment/site/novacustom-V54-series.png)](https://novacustom.com/product/V54-series/)
+
+## Qubes-certified options
+
+The configuration options required for Qubes certification are detailed below.
+
+### Screen size
+
+- Certified: 14 inch
+
+**Note:** The 14-inch model (V540TU) and the 16-inch model (V560TU) are two separate products. [The 16-inch model has already been certified.](/news/2024/09/17/novacustom-v56-series-qubes-certified/)
+
+### Screen resolution
+
+- Certified: Full HD+ (1920 x 1200)
+- Certified: 2.8K (2880 x 1800)
+
+### Processor and graphics
+
+- Certified: Intel Core Ultra 5 Processor 125H, Intel Arc iGPU with AI Boost
+- Certified: Intel Core Ultra 7 Processor 155H, Intel Arc iGPU with AI Boost
+- The Nvidia discrete GPU options are not currently certified.
+
+### Memory
+
+- Certified: Any configuration with at least 16 GB of memory
+
+### Storage
+
+- Certified: All of the available options in these sections
+
+### Personalization
+
+- This section is merely cosmetic and therefore does not affect certification.
+
+### Firmware options
+
+- Qubes OS does not currently support UEFI secure boot.
+- Keeping up-to-date with firmware updates is merely an email notification service and therefore does not affect certification.
+- Deploying coreboot+Heads does not affect certification, but it is not currently an available option for this model anyway.
```

Comment:
Heads variant is a separate firmware variant and as such will require separate certification process. While I do expect it to get certified, it isn't automatic.